discord/state.py

```diff
@@ -454,6 +513,14 @@ def emojis(self) -> Sequence[Emoji]:
     def stickers(self) -> Sequence[GuildSticker]:
         return utils.SequenceProxy(self._stickers.values())
 
+    @property
+    def soundboard_sounds(self) -> Sequence[SoundboardSound]:
+        all_sounds = []
+        for guild in self.guilds:
+            all_sounds.extend(guild.soundboard_sounds)
+
+        return utils.SequenceProxy(all_sounds)
```

Comment:
`SequenceProxy` is there to facilitate copy-on-write. Since you're creating a list flat out there's no reason to use it.